tests/baselines/reference/collisionArgumentsClassConstructor.errors.txt

```diff
@@ -1,40 +1,89 @@
 tests/cases/compiler/collisionArgumentsClassConstructor.ts(3,28): error TS2396: Duplicate identifier 'arguments'. Compiler uses 'arguments' to initialize rest parameters.
+tests/cases/compiler/collisionArgumentsClassConstructor.ts(3,31): error TS1100: Invalid use of 'arguments' in strict mode.

```

Comment:
Can we make the error message a bit better when inside a class.. it is just a hard change for users migrating.